src/Jackett.Common/Indexers/GimmePeers.cs

```diff
@@ -146,7 +134,9 @@ private async Task ProcessPage(List<ReleaseInfo> releases, string searchUrl)
                 var parser = new HtmlParser();
                 var dom = parser.ParseDocument(results);
 
-                var rows = dom.QuerySelectorAll(".browsetable").Last().QuerySelectorAll("tr"); //the class for the table changed
+                var rows = dom.QuerySelectorAll(".browsetable")?.Last().QuerySelectorAll("tr");
```

Comment:
Are you sure of this? I think `dom.QuerySelectorAll(".browsetable")` always returns a list (that can be empty, but not null).

Reply:
Oops, I thought the error log was giving a null reference exception. It's a no elements exception, you're right.